src/main/scala/com/databricks/spark/sql/perf/mllib/MLPipelineStageBenchmarkable.scala

```diff
@@ -72,9 +72,17 @@ class MLPipelineStageBenchmarkable(
       val (scoreTrainTime, scoreTraining) = measureTime {
         test.score(param, trainingData, model)
       }
+      val metricTrainTime = new MLMetrics("trainTime", scoreTrainTime.toMillis, false)
```

Comment:
Let's use "training" instead of "train" (verb). We can use `.` to separate `training`/`test` from the metric names. Say `training.accuracy`, `training.time`, `training.areaUnderROC`.  So we don't need to worry about keep camelCase ...